### PR TITLE
chore(test): update test setup

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

File: tests/utils.ts [hunk 1]
```diff
@@ -93,19 +92,27 @@ export async function deployArIOContract({
...
           purchasePrice: 0,
-          startTimestamp: currentBlock,
+          startTimestamp: currentBlockTimestamp,
           type: 'lease',
           undernames: 10,
         },
         'test-undername': {
           contractTxId: 'I-cxQhfh0Zb9UqQNizC9PiLC41KpUeA9hjiVV02rQRw',
-          endTimestamp: currentBlock + 262800, // 1 year in blocks
+          endTimestamp: currentBlockTimestamp + oneYearMs, // 1 year in blocks
```
Comment (atticusofsparta) [inline tests/utils.ts:113]: OHHHHH.

Im a dumbass >.<

good catch

**Review by atticusofsparta — APPROVED** [review]

File: docker-compose.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   arlocal:
-    image: textury/arlocal:v1.1.35
+    image: textury/arlocal:v1.1.66
```
Comment (dtfiedler) [inline docker-compose.yaml:5]: older versions of arlocal provide timestamps to the milliseconds which throws off our evaluation - newer ones use seconds

**Pull request merged**